src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1386,6 +1387,10 @@ emit_vector64_vector128_t (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSign
 		if (fsig->param_count != 1 )
 			return NULL;
 		return emit_simd_ins_for_unary_op (cfg, klass, fsig, args, arg0_type, id);
+	case SN_op_UnaryPlus:
+		if (fsig->param_count != 1)
+			return NULL;
+		return args [0];
```

Comment:
Does this mean it is a no-op?

Reply:
In the docs it says:

> The unary `+` operator returns the value of its operand.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/arithmetic-operators#unary-plus-and-minus-operators